modules/cluster-logging-collector-pod-location.adoc

```diff
@@ -5,7 +5,7 @@
 [id="cluster-logging-collector-pod-location_{context}"]
 = Viewing logging collector pods
 
-You can use the `oc get pods  --all-namespaces -o wide` command to see the nodes where the Fluentd are deployed.
+You can use the `oc get pods  --all-namespaces -o wide` command to see all the pods and the nodes, and the details about where they are running.
 
 .Procedure
```

Comment:
This single step in the procedure should use an unordered list. Please prepend `*` to the step.

Reply:
Done.